Sources/FunctionalTableData/TableSectionChangeSet.swift

```diff
@@ -11,10 +11,6 @@ import Foundation
 struct Moved<T: Equatable>: Equatable {
 	let from: T
 	let to: T
-
-	static func ==(lhs: Moved<T>, rhs: Moved<T>) -> Bool {
-		return lhs.from == lhs.from && lhs.to == rhs.to
```

Comment:
That `lhs.from == lhs.from`

Reply:
Unsure about a test for this 🤔 how come no test changed

Reply:
I am wondering if it has anything to do with https://github.com/Shopify/store/issues/11412
We have been tracking down this issue that ultimately creates the inconsistent table data exception but the change set is correct. I have to deep dive on how this struct is used.